notebooks/gotcha.clj

```diff
@@ -0,0 +1,27 @@
+(ns gotcha
```

Comment:
I'd prefer `gotchas` to be more consistent w/ the chapter name.

Reply:
done